klaxer/models.py

```diff
@@ -83,7 +97,7 @@ def __repr__(self):
         return '<%s.%s>' % (self.__class__.__name__, self.name)
 
 class Severity(NoValueEnum):
```

Comment:
If we drop the string value, it's probably worth converting to an `IntEnum`. [More info](https://docs.python.org/3/library/enum.html#intenum).